src/main/java/de/qabel/desktop/daemon/sync/worker/index/sqlite/DesktopSyncDatabase.java

```diff
@@ -11,10 +11,22 @@ public DesktopSyncDatabase(Connection connection) {
         super(connection);
     }
 
+    private long version = 0;
+
     @Override
     public AbstractMigration[] getMigrations(Connection connection) {
         return new AbstractMigration[]{
             new Migration1462872507CreateSyncedState(connection)
         };
     }
+
+    @Override
+    public long getVersion() {
+        return version;
```

Comment:
from the interface docblock:
> Read the version of the last executed migration from the database

this is probably the implementation you need (read/write from/to the PRAGMA USER_VERSION):
https://github.com/Qabel/qabel-core/commit/48e3ec5d25e667c0d08746c12ddb348fdc7906d8#diff-37c5d765251f069dffba32c1c1543d78L25